### ISSUE TITLE
As of today when I pull mesosphere/marathon:latest from Docker, I get the wrong image.

### ISSUE THREAD

**Issue by dpgeekzero** [mesosphere/marathon#4509]
I'm sure this probably an issue with the docker hub, but as Marathon is affected, I thought I'd mention it here.

```
doug@dougdev ~> docker pull mesosphere/marathon                                                                                                                                                                                             1
Using default tag: latest
latest: Pulling from mesosphere/marathon

16049eee30ce: Pull complete 
bffcc41827c8: Pull complete 
8db5432a6ca1: Pull complete 
7fb68ebbf612: Pull complete 
204f231daf70: Pull complete 
136682d7a30e: Pull complete 
b714021e028a: Pull complete 
7cac8a13dff2: Pull complete 
Digest: sha256:da65edd5cbcfddebd54e2450c12596ea2e5718be92dae0776ffcc87e9cc14b1c
Status: Downloaded newer image for mesosphere/marathon:latest
doug@dougdev ~> docker run -it mesosphere/marathon 
Set 'mysql.host' in local configuration.
Set 'mysql.port' in local configuration.
Set 'mysql.user' in local configuration.
Set 'phabricator.base-uri' in local configuration.
Set 'pygments.enabled' in local configuration.
Set 'repository.default-local-path' in local configuration.
Set 'storage.local-disk.path' in local configuration.
MySQL Credentials Not Configured

Unable to connect to MySQL using the configured credentials. You must
configure standard credentials before you can upgrade storage. Run these
commands to set up credentials:

  phabricator/ $ ./bin/config set mysql.host __host__
  phabricator/ $ ./bin/config set mysql.user __username__
  phabricator/ $ ./bin/config set mysql.pass __password__

These standard credentials are separate from any administrative credentials
provided to this command with __--user__ or __--password__, and must be
configured correctly before you can proceed.

Raw MySQL Error: Attempt to connect to root@localhost:3306 failed with error
#2002: No such file or directory.
Updating nginx server_name to phabricator.local
Updating php configuration
[2016-09-28 13:08:59] EXCEPTION: (AphrontConnectionQueryException) Attempt to connect to root@localhost:3306 failed with error #2002: No such file or directory. at [<phutil>/src/aphront/storage/connection/mysql/AphrontBaseMySQLDatabaseConnection.php:343]
arcanist(head=stable, ref.stable=10e519475290), phabricator(head=stable, ref.stable=25eea83bc05f), phutil(head=stable, ref.stable=fc84916a1a31)
  #0 AphrontBaseMySQLDatabaseConnection::throwConnectionException(integer, string, string, string) called at [<phutil>/src/aphront/storage/connection/mysql/AphrontMySQLDatabaseConnection.php:82]
  #1 AphrontMySQLDatabaseConnection::connect() called at [<phutil>/src/aphront/storage/connection/mysql/AphrontBaseMySQLDatabaseConnection.php:101]
  #2 AphrontBaseMySQLDatabaseConnection::establishConnection() called at [<phutil>/src/aphront/storage/connection/mysql/AphrontBaseMySQLDatabaseConnection.php:124]
  #3 AphrontBaseMySQLDatabaseConnection::requireConnection() called at [<phutil>/src/aphront/storage/connection/mysql/AphrontMySQLDatabaseConnection.php:12]
  #4 AphrontMySQLDatabaseConnection::escapeBinaryString(string) called at [<phutil>/src/aphront/storage/connection/mysql/AphrontMySQLDatabaseConnection.php:8]
  #5 AphrontMySQLDatabaseConnection::escapeUTF8String(string) called at [<phutil>/src/xsprintf/qsprintf.php:178]
  #6 xsprintf_query(AphrontMySQLDatabaseConnection, string, integer, string, integer) called at [<phutil>/src/xsprintf/xsprintf.php:70]
  #7 xsprintf(string, AphrontMySQLDatabaseConnection, array) called at [<phutil>/src/xsprintf/qsprintf.php:64]
  #8 qsprintf(AphrontMySQLDatabaseConnection, string, string, string, string)
  #9 call_user_func_array(string, array) called at [<phutil>/src/xsprintf/queryfx.php:5]
  #10 queryfx(AphrontMySQLDatabaseConnection, string, string, string, string)
  #11 call_user_func_array(string, array) called at [<phutil>/src/xsprintf/queryfx.php:13]
  #12 queryfx_all(AphrontMySQLDatabaseConnection, string, string, string, string)
  #13 call_user_func_array(string, array) called at [<phutil>/src/aphront/storage/connection/AphrontDatabaseConnection.php:36]
  #14 AphrontDatabaseConnection::queryData(string, string, string, string)
  #15 call_user_func_array(array, array) called at [<phabricator>/src/infrastructure/storage/lisk/LiskDAO.php:535]
  #16 LiskDAO::loadRawDataWhere(string, string)
  #17 call_user_func_array(array, array) called at [<phabricator>/src/infrastructure/storage/lisk/LiskDAO.php:476]
  #18 LiskDAO::loadAllWhere(string, string) called at [<phabricator>/src/infrastructure/env/PhabricatorConfigDatabaseSource.php:19]
  #19 PhabricatorConfigDatabaseSource::loadConfig(string) called at [<phabricator>/src/infrastructure/env/PhabricatorConfigDatabaseSource.php:7]
  #20 PhabricatorConfigDatabaseSource::__construct(string) called at [<phabricator>/src/infrastructure/env/PhabricatorEnv.php:232]
  #21 PhabricatorEnv::buildConfigurationSourceStack(boolean) called at [<phabricator>/src/infrastructure/env/PhabricatorEnv.php:95]
  #22 PhabricatorEnv::initializeCommonEnvironment(boolean) called at [<phabricator>/src/infrastructure/env/PhabricatorEnv.php:75]
  #23 PhabricatorEnv::initializeScriptEnvironment(boolean) called at [<phabricator>/scripts/init/lib.php:22]
  #24 init_phabricator_script(array) called at [<phabricator>/scripts/init/init-script.php:10]
  #25 require_once(string) called at [<phabricator>/scripts/__init_script__.php:3]
  #26 require_once(string) called at [<phabricator>/scripts/daemon/manage_daemons.php:5]
2016-09-28 13:08:59,661 CRIT Supervisor running as root (no user in config file)
2016-09-28 13:08:59,661 CRIT Supervisor running as root (no user in config file)
2016-09-28 13:08:59,662 INFO supervisord started with pid 1
2016-09-28 13:08:59,662 INFO supervisord started with pid 1
2016-09-28 13:09:00,664 INFO spawned: 'nginx' with pid 23
2016-09-28 13:09:00,664 INFO spawned: 'nginx' with pid 23
2016-09-28 13:09:00,665 INFO spawned: 'php-cgi' with pid 24
2016-09-28 13:09:00,665 INFO spawned: 'php-cgi' with pid 24
2016-09-28 13:09:00,665 INFO reaped unknown pid 20
2016-09-28 13:09:00,665 INFO reaped unknown pid 20
nginx: [warn] "ssl_stapling" ignored, issuer certificate not found
2016/09/28 13:09:00 [notice] 23#0: using the "epoll" event method
2016/09/28 13:09:00 [notice] 23#0: nginx/1.10.1
2016/09/28 13:09:00 [notice] 23#0: OS: Linux 4.7.4-1-ARCH
2016/09/28 13:09:00 [notice] 23#0: getrlimit(RLIMIT_NOFILE): 65536:65536
2016/09/28 13:09:00 [notice] 23#0: start worker processes
2016/09/28 13:09:00 [notice] 23#0: start worker process 25
[28-Sep-2016 13:09:00] WARNING: Nothing matches the include pattern '/etc/php5/fpm.d/*.conf' from /etc/php5/php-fpm.conf at line 15.
^C2016-09-28 13:09:01,266 WARN received SIGINT indicating exit request
2016-09-28 13:09:01,266 WARN received SIGINT indicating exit request
2016-09-28 13:09:01,267 INFO waiting for nginx, php-cgi to die
2016-09-28 13:09:01,267 INFO waiting for nginx, php-cgi to die
2016-09-28 13:09:01,270 INFO stopped: php-cgi (exit status 0)
2016-09-28 13:09:01,270 INFO stopped: php-cgi (exit status 0)
2016/09/28 13:09:01 [notice] 23#0: signal 15 (SIGTERM) received, exiting
2016/09/28 13:09:01 [notice] 25#0: exiting
2016/09/28 13:09:01 [notice] 25#0: exit
2016/09/28 13:09:01 [notice] 23#0: signal 17 (SIGCHLD) received
2016/09/28 13:09:01 [notice] 23#0: worker process 25 exited with code 0
2016/09/28 13:09:01 [notice] 23#0: exit
2016-09-28 13:09:01,316 INFO stopped: nginx (exit status 0)
2016-09-28 13:09:01,316 INFO stopped: nginx (exit status 0
```

**Comment by jasongilanfarr** [comment]
We're investigating, I accidentally mistyped the image name and pushed the wrong one.

**Comment by jasongilanfarr** [comment]
In the meantime, can you use marathon:v1.3.0 as this is the most recent release?

**Comment by dpgeekzero** [comment]
I already had it cached, so I just retagged what I had as latest and moved on.

Thanks for the quick response.

**Comment by jasongilanfarr** [comment]
The latest tag has been fixed to point back at v1.3.0
